apps/api/src/applicant/external-api.service.ts

```diff
@@ -294,13 +302,16 @@ export class ExternalAPIService {
     id: number | undefined = undefined,
     is_success = false,
     additional_data: object | undefined | unknown = undefined,
+    manager?: EntityManager,
   ): Promise<SyncApplicantsAudit> {
     if (id) {
       const audit = await this.syncApplicantsAuditRepository.findOne(id);
       if (audit) {
         audit.is_success = is_success;
         audit.additional_data = additional_data;
-        await this.syncApplicantsAuditRepository.save(audit);
+        manager
```

Comment:
we can always pass manager.

Reply:
saveSyncApplicantsAudit gets called in the catch block of the try/catch, outside the transaction, I suppose we could re-organize it or just remove that catch call?

Reply:
You're right. we can use `finally` block to log the result instead of `catch`.